README.md

```diff
@@ -4,7 +4,7 @@
 
 # Chromatic
 
-🍪 Configurable human assisted Chrome automation
+🍪 Chromatic is a tool for Configurable human assisted Chrome automation.
```

Comment:
The capitalization needs to be fixed, and it doesn't read _quite_ right. How about:

> 🍪 A tool for configurable, human-assisted Chrome automation

Having "Chromatic" at the start of the line _and_ as the header above, stutters a little.